openqaoa/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.1dev"
\ No newline at end of file
+__version__ = "0.0.2"
```

Comment:
we were on "0.0.1-beta" previously, so shouldn't this be "0.0.1"?

Reply:
The version will most likely crash with the PyPI test until we reach 0.0.5 :D

Reply:
But you are right, I will change this PR to 1 and then we see